setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml

```diff
@@ -15,45 +14,13 @@
   with_items:
     - git
     - build-essential
-    - python
-    - python-setuptools
-    - python-pip
-    - python-virtualenv
     - cmake
-    - swig
     - wget
-    - cmake
```

Comment:
![screen shot 2015-05-06 at 11 17 16 am](https://cloud.githubusercontent.com/assets/60851/7500280/81f4d55c-f3e1-11e4-9ed0-54dc4636feef.png)

That extra one is not meaningful, right?